docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -0,0 +1,465 @@
+# Component Port
+
+Tizen components of the component-based applications can communicate with each other using component ports. Components can send and receive serializable objects through component port communication.
+
+The main feature of the `Tizen.Applications.ComponentBased.ComponentPort` class include:
+- Managing component port
+  You can set up the component ports to [send and receive requests](#port) between components of component-based application with the `Tizen.Applications.ComponentBased.ComponentPort` class.
+
+- Using component task
+  You can set up the component tasks to [run tasks](#task) and wait for events from other components with the `Tizen.Applications.ComponentBased.ComponentTask` class.
```

Comment:
All is well, but we need one line space between: 

6-7 
9-10 

By including one line space, the formatting will be as desired.

Reply:
Done